charts/testrunnerdashboard/templates/ingress/service.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.ingress.enabled -}}
```

Comment:
@knapii-developments what are your thoughts here? Is this something where we should start off using the ingress library you've been working on from the start (there's a few other ingress files here, too)?

Reply:
@jack-r-warren @knapii-developments The ingress has been refactored to use `ingresslib`. Please have a look and let me know how it goes. Thanks!